website/docs/en/guide/migration/umi.mdx

```diff
@@ -0,0 +1,77 @@
+# Umi
+
+This chapter introduces how to migrate [Umi](https://umijs.org/docs/introduce/introduce) project to Rsbuild.
+
+**Notice: Make sure your Umi version is >= 4.0**
+
+## Installation
+
+```bash
+npm add @aumi/aumi@latest -S
+```
+
+[`@aumi/aumi`](https://github.com/atom-yang/aumi) replaces Umi's default bundler from Webpack to Rsbuild, it is possible to improve build performance while retaining Umi's build system
```

Comment:
Thank you so much! We really appreciate your work ❤️

Since `@aumi/aumi` is a community-driven project, we prefer to maintain this migration guide in the aumi repository, and  we can add aumi to https://github.com/web-infra-dev/awesome-rspack, so that more users can find and use it.

Reply:
Got it, will updating `@aumi/aumi` docs and closing this PR, thanks.

Reply:
Do you want to submit a PR to https://github.com/web-infra-dev/awesome-rspack? If not, we can help to add it.

Reply:
Of course，will submit today's later.

Reply:
Nice 👍